### DISCUSSION TITLE
Can Onboarding be done only for User Model?

### DISCUSSION THREAD

**Discussion by abishekrsrikaanth** [spatie/laravel-onboard#18] [Q&A]
If my onboarding process wasn't attached to the User but to a Site/Domain on a SaaS app, can I implement the `Onboardable` interface and `GetsOnboarded` trait on the model that requires the onboarding? And also do the steps need to added only on the AppServiceProvider? Can it be added on a controller that loads the view of the onboarding page?

**Comment by riasvdv** [comment]
Onboarding can be added to any model, we use it for a Team model for example.
